docs/content/010-getting-started/03-tutorial/06-chapter-5-persisting-data-via-prisma.mdx

```diff
@@ -223,15 +222,16 @@ export function createContext(): Context {
 
 ```ts
 // api/context.ts
-import { PrismaClient } from '@prisma/client'
+import { db } from "./db";
+import { PrismaClient } from "@prisma/client"
 
 export interface Context {
   db: PrismaClient
 }
 
 export function createContext(): Context {
   return {
-    db: new PrismaClient(),
+    db,
```

Comment:
Can you update this to match https://github.com/graphql-nexus/tutorial/blob/master/api/context.ts

I don't think we need a db modular just to instantiate a Prisma client :)

Reply:
We don't need db to modular, but we do need to store the instance of the Prisma Client in either an object or variable which is exported so that it can be accessed in `tests/__helpers.ts`. The same Prisma Client used in the context of Nexus should be accessible in `__helpers.ts` as a new connection is created for every instance of Prisma Client. 

There are two options, one is keep it as I've updated it.

Another is, to export an object from 'context.ts' and disconnect the connection of that
```ts
export const context: Context = {
  db: new PrismaClient()
}
```
and in `__helpers.ts`
```ts
import { context } from '../api/context.ts'

... 

function graphqlTestContext() {
  ...
  
  return {
    async before() {
      ...
      serverInstance.server.on('close', async () => {
        await context.db.$disconnect()
      })
    },
    ...
  }
}
```

Reply:
Yep good point. then a singleton db module is a simple solution indeed. If you're inclined update the tutorial repo otherwise we'll try to do it soon.

Reply:
Already done graphql-nexus/tutorial#13 🙂 !